cli/lsp/language_server.rs

```diff
@@ -1331,6 +1386,8 @@ impl Inner {
         self.config.set_workspace_settings(settings, vec![]);
       }
     };
+    // TODO(nathanwhit): allow updating after startup, needs work to set thread local collector on tsc thread
```

Comment:
Do you want to actually address this? If not, then open an issue so it doesn't get lost